models/project/project.go

```diff
@@ -161,20 +161,20 @@ func (u *ProjectStruct) Initialize() {
 	u.database = interfaceDB
 }
 
-func (u *ProjectStruct) Save() bool {
+func (u *ProjectStruct) Save() error {
 	table := u.database.Init()
 	defer table.Close()
 
 	result, err := table.Exec("insert into projects (user_id, repository_id, title, description, show_issues, show_pull_requests, created_at) values (?, ?, ?, ?, ?, ?, now());", u.UserID, u.RepositoryID, u.Title, u.Description, u.ShowIssues, u.ShowPullRequests)
 	if err != nil {
 		logging.SharedInstance().MethodInfo("Project", "Save", true).Errorf("failed to save project: %v", err)
-		return false
+		return err
 	}
 	u.ID, _ = result.LastInsertId()
-	return true
+	return nil
 }
 
-func (u *ProjectStruct) Update(title string, description string, showIssues bool, showPullRequests bool) bool {
+func (u *ProjectStruct) Update(title string, description string, showIssues bool, showPullRequests bool) error {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.Update should have comment or be unexported